aptos-move/block-executor/src/view.rs

```diff
@@ -1867,84 +1867,40 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> ValueToIden
         kind: &IdentifierMappingKind,
         layout: &MoveTypeLayout,
         value: Value,
-    ) -> TransformationResult<Value> {
+    ) -> PartialVMResult<SizedID> {
```

Comment:
So, we are using the name "DelayedFieldID" in some places and "SizedID" in other places. What's the difference? Why not keep the old name "DelayedFieldID" everywhere?

Reply:
SizedID lives in Move third party and originally it was named like that because I though we would keep `ValueImpl::Native` in a generic way. But this seems no longer to be the case, so I will do a pass and rename it (see slack discussion about it)

Reply:
I ll probably wait until @gelash finishes his pass before pushing that😂

Reply:
From your above comments, my understanding is that `SizedID = (DelayedFieldID, #bytes)`. Is that accurate?

Reply:
(as mentioned somewhere else, `SizedID` is gone in favour of `DelayedFieldID`). But previously, `SizedID = DelayedFieldID` because @igor-aptos recently changed `DelayedFieldID` to carry an identifier (u32) and a size of actual field we replaced (called `width: u32`)